lib/text_helpers.js

```diff
@@ -32,11 +32,12 @@ function paginateMessage(message, textToSend) {
  * Get a result limit from the specified message, constraining it to values
  * that aren't obscene.
  *
- * @param {Object} message - Discord.js Message object from which to get value
+ * @param {string} message - The message
+ * @param {number} defaultLimit - A default limit to use if no value is set
  * @return {number} Constrained limit from the message
  */
-function getLimitFromMessage(message) {
-  let limit = 10;
+function getLimitFromMessage(message, defaultLimit) {
```

Comment:
You've not updated `command_handlers/user_stats.js` to reflect this change. Either add a default value for the parameter (`defaultLimit = 10`) or update all callers.